src/watchmaker/workers/salt.py

```diff
@@ -669,6 +674,7 @@ def __init__(self, *args, **kwargs):
         # Pop arguments used by SaltWindows
         self.installer_url = kwargs.pop('installer_url', None) or ''
         self.ash_role = kwargs.pop('ash_role', None) or ''
+        self.admin_groups = kwargs.pop('admin_groups', None) or ''
```

Comment:
This ought to be moved after the `super(...)`, so that the SaltBase init does not set the value back to `None`.